objdiff.json

```diff
@@ -1,44 +1,37 @@
 {
-    "custom_make": "python",
```

Comment:
This updated objdiff to the latest version, which is incompatible with the objdiff used in CI ^^